cloud-storage-uploader/README.md

```diff
@@ -0,0 +1,194 @@
+<!--
+Copyright 2019 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# cloud-storage-uploader
+
+This action uploads files/folders to a [GCS][gcs] bucket. This is useful when
+you want upload build artifacts from your workflow.
+
+Paths to files that are successfully uploaded are set as output variables and can be
+used in subsequent steps.
+
+## Prerequisites
+
+- This action requires Google Cloud credentials that are authorized to upload
+  blobs to the specified bucket. See the Authorization section below for more
+  information.
+
+## Usage
+
+### For uploading a file
+
+```yaml
+steps:
+  - id: upload-file
+    uses: GoogleCloudPlatform/github-actions/cloud-storage-uploader@master
```

Comment:
The current actions have verb-name (e.g. setup-gcloud, get-secretmanager-secrets). Should this be `upload-cloud-storage`?

Reply:
renamed action to `upload-cloud-storage`